### PR TITLE
testing/rtrlib: new aport

### PR DESCRIPTION
**Pull request by cfra**
https://github.com/rtrlib/rtrlib
An open-source C implementation of the RPKI/Router Protocol client

### CODE REVIEW

File: testing/rtrlib/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Maintainer: Christian Franke <nobody@nowhere.ws>
+pkgname=rtrlib
+pkgver=0.6.3
+pkgrel=0
+pkgdesc="An open-source C implementation of the RPKI/Router Protocol client"
+url="https://github.com/rtrlib/rtrlib"
+arch="all"
+license="MIT"
+depends=""
```
Comment (andypost) [inline testing/rtrlib/APKBUILD]: no reason to keep empty lines

File: testing/rtrlib/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+makedepends="cmake cmocka-dev doxygen graphviz libssh-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/rtrlib/$pkgname/archive/v$pkgver.tar.gz"
```
Comment (andypost) [inline testing/rtrlib/APKBUILD]: better to remove `$pkgname` from URL to easy navigate to source

File: testing/rtrlib/APKBUILD (hunk 1 above)
Comment (cfra) [reply testing/rtrlib/APKBUILD]: Good point, thank you for pointing it out. :pray:

File: testing/rtrlib/APKBUILD (hunk 2 above)
Comment (cfra) [reply testing/rtrlib/APKBUILD]: Done.

**Comment by cfra** [comment]
@andypost I think I have addressed both of your coments.

Is there anything else which I should do to move this PR further along?

**Pull request closed without merging**